rocAL/include/pipeline/enum_registry.h

```diff
@@ -76,6 +85,22 @@ class EnumRegistry {
         return (it != _enum_map.end()) ? it->second : "";
     }
 
+    /*!
+     * \brief Convert integer value to enum and return as std::any
+     * \param enumTypeName The name of the enum type
+     * \param value The integer value to convert
+     * \return The converted enum value wrapped in std::any
+     * \throws std::runtime_error if enum type is not registered
+     */
+    std::any convertIntToEnum(const std::string& enumTypeName, int value) const {
+        auto it = _name_to_converter_map.find(enumTypeName);
+        if (it != _name_to_converter_map.end()) {
+            return it->second(value);  // Call the conversion function and return std::any
+        } else {
+            throw std::runtime_error("No conversion function registered for enum: " + enumTypeName);
```

Comment:
Blocking: `std::runtime_error` here requires including `<stdexcept>`. Right now this header fails to compile if it is included on its own (g++ errors with "std::runtime_error" not found). Please add the missing include.